### PR TITLE
Fix part of #41: Hifi Story List 

### PR DESCRIPTION
**Pull request by Luffy18346**
## Explanation
Fix part of #41: Changes UI of Story header and story list item in StoryList

## Checklist

- [x] The PR title starts with "Fix #bugnum: ", followed by a short, clear summary of the changes. (If this PR fixes part of an issue, prefix the title with "Fix part of #bugnum: ...".)
- [x] The PR explanation includes the words "Fixes #bugnum: ..." (or "Fixes part of #bugnum" if the PR only partially fixes an issue).
- [x] The PR follows the [style guide](https://github.com/oppia/oppia-android/wiki/Coding-style-guide).
- [x] The PR does not contain any unnecessary auto-generated code from Android Studio.
- [x] The PR is made from a branch that's **not** called "develop".
- [x] The PR is made from a branch that is up-to-date with "develop".
- [x] The PR's branch is based on "develop" and not on any other branch.
- [ ] The PR is **assigned** to an appropriate reviewer in both the **Assignees** and the **Reviewers** sections.

### CODE REVIEW

File: app/src/main/res/layout/story_chapter_view.xml [hunk 1]
```diff
@@ -76,6 +73,7 @@
             android:layout_width="0dp"
             android:layout_height="wrap_content"
             android:layout_marginStart="8dp"
+            android:layout_marginTop="4dp"
             android:layout_marginEnd="8dp"
             android:fontFamily="sans-serif-medium"
```
Comment (rt4914) [inline app/src/main/res/layout/story_chapter_view.xml:79]: Add these attributes to `chapter_title`
```
android:ellipsize="end"
android:maxLines="2"
```

Add these attributes to chapter_summary

```
android:ellipsize="end"
android:maxLines="4"
```

File: app/src/main/res/layout/story_chapter_view.xml [hunk 2]
```diff
@@ -41,10 +41,7 @@
           android:id="@+id/chapter_thumbnail"
           android:layout_width="match_parent"
           android:layout_height="wrap_content"
-          android:contentDescription="@{viewModel.name}"
-          android:paddingStart="40dp"
-          android:paddingEnd="40dp"
-          android:scaleType="centerCrop"
+          android:scaleType="centerInside"
           android:src="@{viewModel.chapterThumbnail.thumbnailGraphic}"
           app:layout_constraintDimensionRatio="H,4:3"
```
Comment (rt4914) [inline app/src/main/res/layout/story_chapter_view.xml]: I am not sure about this ratio here, let me confirm on this.

**Review by rt4914 — CHANGES_REQUESTED** [review]
@Luffy18346 PTAL

**Comment by rt4914** [comment]
@Luffy18346 please add the final screenshot of your work in the description of the PR.

File: app/src/main/res/layout/story_chapter_view.xml (hunk 2 above)
Comment (rt4914) [reply app/src/main/res/layout/story_chapter_view.xml]: The ratio here should be `16:9`

**Comment by rt4914** [comment]
@Luffy18346 One more change, the tick mark used in develop code is different than the tick mark in mocks.

Update the tick mark in `chapter_completed_tick`. You can use tick mark from android-assets.

File: app/src/main/res/layout/story_chapter_view.xml (hunk 2 above)
Comment (Luffy18346) [reply app/src/main/res/layout/story_chapter_view.xml]: I am also not sure about that. I have sent u updated screens on Hangout. PTAL at 3rd screen, the size of image is confusing me compared to first 2 screens. I think we have to put some restriction on height.

Comment (rt4914) [reply app/src/main/res/layout/story_chapter_view.xml]: @Luffy18346 Please add those screenshots here for everyone's reference. Also the ratio is 16:9.

File: app/src/main/res/layout/story_chapter_view.xml (hunk 1 above)
Comment (Luffy18346) [reply app/src/main/res/layout/story_chapter_view.xml:79]: @rt4914 , all above suggested changes are done. Tick mark icon is also updated as per shown in mock.

**Comment by Luffy18346** [comment]
These are the final screenshots of the work

updateImage1


updatedImage2


updatedImage3

**Comment by Luffy18346** [comment]
These are final screenshots of the work 

updateImage1


updateImage2


updateImage3

File: app/src/main/res/layout/story_chapter_view.xml (hunk 2 above)
Comment (Luffy18346) [reply app/src/main/res/layout/story_chapter_view.xml]: @rt4914 , I have updated the ratio to 16:9 and also added final screenshots or the work at the bottom

**Review by rt4914 — CHANGES_REQUESTED** [review]
Just one more change, remove the unused `ic_check_primary` icon.

File: .idea/misc.xml [hunk 3]
```diff
@@ -6,4 +6,4 @@
   <component name="ProjectType">
     <option name="id" value="Android" />
   </component>
-</project>
+</project>
```
Comment (rt4914) [inline .idea/misc.xml]: Revert these file changes.

File: .idea/inspectionProfiles/Project_Default.xml [hunk 4]
```diff
@@ -0,0 +1,6 @@
+<component name="InspectionProjectProfileManager">
+  <profile version="1.0">
+    <option name="myName" value="Project Default" />
+    <inspection_tool class="IncorrectScope" enabled="false" level="ERROR" enabled_by_default="false" />
+  </profile>
+</component>
```
Comment (rt4914) [inline .idea/inspectionProfiles/Project_Default.xml]: Revert these file changes.

File: .idea/codeStyles/Project.xml [hunk 5]
```diff
@@ -157,4 +162,4 @@
       </indentOptions>
     </codeStyleSettings>
   </code_scheme>
-</component>
+</component>
```
Comment (rt4914) [inline .idea/codeStyles/Project.xml]: Revert these file changes.

File: app/src/main/res/layout/story_chapter_view.xml (hunk 1 above)
Comment (rt4914) [reply app/src/main/res/layout/story_chapter_view.xml:79]: Thanks.

**Review by rt4914 — CHANGES_REQUESTED** [review]
Please remove auto-changed code.

**Comment by rt4914** [comment]
@Luffy18346 always make sure that before your PR gets closed or merged you have replied to all the conversations and resolved all conversations.

**Review by rt4914 — APPROVED** [review]
LGTM, thanks.

File: .idea/misc.xml (hunk 3 above)
Comment (Luffy18346) [reply .idea/misc.xml]: Changes reverted in this file.

File: .idea/inspectionProfiles/Project_Default.xml (hunk 4 above)
Comment (Luffy18346) [reply .idea/inspectionProfiles/Project_Default.xml]: This file is removed.

File: .idea/codeStyles/Project.xml (hunk 5 above)
Comment (Luffy18346) [reply .idea/codeStyles/Project.xml]: Changes reverted in this file.

**Pull request merged**